### PR TITLE
[IMP] point_of_sale: payment form revamp

### PR DESCRIPTION
**Pull request by jipr-odoo**
In this commit:
================
This commit rearranges several fields in the payment form to follow a standard sequence.

task-3989866

### CODE REVIEW

File: addons/point_of_sale/views/pos_payment_views.xml [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
                         <field name="currency_id" invisible="1" />
-                        <field name="name" />
-                        <field name="amount" />
-                        <field name="pos_order_id" readonly="1"/>
-                        <field name="payment_method_id" readonly="1"/>
-                        <field name="card_type" readonly="1" invisible="not card_type"/>
-                        <field name="cardholder_name" readonly="1" invisible="not cardholder_name"/>
-                        <field name="transaction_id" readonly="1" invisible="not transaction_id"/>
-                        <field name="session_id" />
+                        <field name="name" invisible="1"/>
```
Comment (adgu-odoo) [inline addons/point_of_sale/views/pos_payment_views.xml:11]: Do we need this line ? And do we need the line n°10 ?

**Review by adgu-odoo — CHANGES_REQUESTED** [review]
Can you just check my comment and tell me what you think. After that, it will be ready to r+

File: addons/point_of_sale/views/pos_payment_views.xml (hunk 1 above)
Comment (jipr-odoo) [reply addons/point_of_sale/views/pos_payment_views.xml:11]: yes we can remove both lines since it is always invisible

**Comment by jipr-odoo** [comment]
> Can you just check my comment and tell me what you think. After that, it will be ready to r+

Hello @adgu-odoo build is ready, can you please check again?
Thanks!

**Comment by adgu-odoo** [comment]
@robodoo r+

**Pull request closed without merging**